src/app/modules/main/communities/module.nim

```diff
@@ -164,6 +164,8 @@ method communityDataLoaded*(self: Module) =
   self.buildTokensAndCollectiblesFromAllCommunities()
 
 method onActivated*(self: Module) =
+  if self.curatedCommunitiesLoaded:
```

Comment:
Seems logical. Just one concern: will curated communities be refreshed when a new community is added to or removed from the directory? I suppose this potential issue is negligible, as a restart should resolve it.

Reply:
I'm pretty sure it won't update, but indeed a restart will fix it.

Anyway, the way it was done was not the proper way. We should instead receive a signal and add the new item to the model that way instead of refreshing the model each time we switch to the section.